cli/pcluster/models/hit/hit_cluster_model.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+from botocore.exceptions import ClientError
+
+from pcluster.cluster_model import ClusterModel
+from pcluster.config import mappings
+from pcluster.utils import get_instance_type
+
+
+class HITClusterModel(ClusterModel):
+    """HIT cluster model."""
```

Comment:
In the comment I'd write: Heterogeneous Instance Type model, just to have a point in the code on which we're explaining the acronym :D (I'll do the same for SIT)